airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -50,5 +50,11 @@
     "required": ["account_id", "access_token", "start_date"]
   },
   "supportsIncremental": true,
-  "supported_destination_sync_modes": ["append"]
+  "supported_destination_sync_modes": ["append"],
+  "authSpecification": {
+    "auth_type": "oauth2.0",
+    "oauth2Specification": {
+      "oauthFlowInitParameters": [["access_token"]]
```

Comment:
it should be `[["access_token"], ["access_token"]]`.
You can read about oauthFlowInitParameters here:
https://github.com/airbytehq/airbyte/blob/3378ba430feb5d994ccda71821de89382f37e7a0/airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml#L216